### PR TITLE
Collections && list.txt - HackerRank

### PR DESCRIPTION
**Pull request by mike-angek96**

### CODE REVIEW

File: MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/ArrayListExercise.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			myList.add(mySubListOfNumbers);
+		}
+
+		int queries = SCAN.nextInt();
+		int count = 0;
+		while (count < queries) {
+			int numbers = SCAN.nextInt() - 1;
+			int position = SCAN.nextInt() - 1;
+
+			System.out.println((myList.size() <= numbers || myList.get(numbers).size() <= position ? "ERROR!"
```
Comment (luisod78) [inline MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/ArrayListExercise.java:33]: very good

File: MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/HashSetExercise.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.Scanner;
+import java.util.HashSet;
+import java.util.Set;
+
+public class HashSetExercise {
+	public static final Scanner SCAN = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int numberOfPairs = SCAN.nextInt();
+		String[] pair_left = new String[numberOfPairs];
```
Comment (luisod78) [inline MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/HashSetExercise.java:12]: Local variable names must be written in lowerCamelCase
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

File: MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/SortExercise.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+			String firstName = SCANNER.next();
+			double cgpa = SCANNER.nextDouble();
+
+			StudentSortExercise student = new StudentSortExercise(id, firstName, cgpa);
+			studentList.add(student);
+
+			testCases--;
+		}
+
+		Collections.sort(studentList,
```
Comment (luisod78) [inline MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/SortExercise.java:27]: nice!

File: MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/StudentSortExercise.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package collections;
+
+public class StudentSortExercise {
+	private int id;
+	private String firstName;
+	private double cgpa;
+
+	public StudentSortExercise(int id, String firstName, double cgpa) {
+		super();
```
Comment (luisod78) [inline MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/StudentSortExercise.java:9]: Why calling super() ???
Do you need it?

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/HashSetExercise.java (hunk 2 above)
Comment (mike-angek96) [reply MiguelAngelVidalLiborio/Dia6-30Agosto/hackerrank/collections/HashSetExercise.java:12]: i apologize for this mistake

**Review by luisod78 — APPROVED** [review]

**Review by edbermeo — APPROVED** [review]
LGTM!

**Review by globant-bootcamp — APPROVED** [review]

**Pull request merged**